picard/track.py

```diff
@@ -368,6 +375,23 @@ def _convert_folksonomy_tags_to_genre(self):
             join_with=config.setting['join_genres']
         )
 
+    def _add_tags(self, tags, name):
+        self.metadata[name] = tags.keys()
+
+    def _add_folksonomy_tags(self):
+        tags = Counter(self._folksonomy_tags)
+        tags += self.album._folksonomy_tags
+        self._add_tags(tags, '_folksonomy_tags')
+
+    def _add_genres(self):
+        genre = Counter(self._genres)
+        genre += self.album._genres
+        self._add_tags(genre, '_genres')
+
+    def _delete_genres_from_tags(self):
+        for genres in self.album.genres:
+            del self.album._folksonomy_tags[genres]
```

Comment:
This is a really minor nit-picking comment, but should the variable be `genre` rather than `genres`?

```suggestion
    def _delete_genres_from_tags(self):
        for genre in self.album.genres:
            del self.album._folksonomy_tags[genre]
```

Reply:
Oops. Thanks for pointing it out! Fixed in commit: 3f0a795d8bbceba4d239480a9d38d64cc1bd237e